examples/with-apollo-and-redux/lib/initApollo.js

```diff
@@ -0,0 +1,36 @@
+import { ApolloClient, createNetworkInterface } from 'react-apollo'
+import fetch from 'isomorphic-fetch'
+
+let apolloClient = null
+
+// Polyfill fetch() on the server (used by apollo-client)
+if (!process.browser) {
```

Comment:
Yeah I prefer it being explicit too which is why i don't like it being a global (when it's not _always_ a global). So I prefer to import fetch where it's being used.

But don't let me stop you, it looks good I'm alright with whichever, just wanted to point it out as I've stumbled into this myself a few times.